internal/operatorcli/operator_cli.go

```diff
@@ -287,7 +312,7 @@ func RunRegister(c *cli.Context) error {
 }
 
 func RunDeregister(c *cli.Context) error {
-	passphrase := c.String(flag.PassphraseFlag.Name)
+	ecdsaPassphrase := c.String(flag.EcdsaPassphraseFlag.Name)
```

Comment:
this should only override the general passphrase if the dedicated one is not provided, like the encrypt command works no?

Reply:
since deregister doesn't require more than one password i've left it as it is, but made it explicit with ecdsa passphrase flag that it is ecdsa key password

Reply:
we should be backward compatibility, so we should pass the two passphrase flags use the backward compatibility flag if the correct flag is empty